pkg/runner/local_runner.go

```diff
@@ -136,6 +157,44 @@ func (r *localRunner) Run(ctx context.Context, request *runner.RunRequest) (*run
 	}
 	cmd.Stdout = stdout
 
+	// Mount special filesystems.
+	requiresSpecialFilesystems := cmd.SysProcAttr != nil && cmd.SysProcAttr.Chroot != ""
+	var MountRootDirectory filesystem.Directory
+
+	if requiresSpecialFilesystems {
+		var dir filesystem.Directory
+		dir = r.buildDirectory
+		// TODO(nils): How to best open the input root `Directory`
+		// It must be a `localDirectory`, but outer directories can be `lazyDirectory`.
```

Comment:
This strings-split is a cop out for my lack of understanding of the builder and scope walker.
I want to replace this block.

:-1:

Reply:
No worries. I can also help out with this part, after we get the bb-storage part sorted out.